playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -55,3 +56,35 @@ test('S487 C651123 open keyboard shortcut modal using hot keys', async ({ page }
   await expect(ksModal.ksModal).toBeHidden()
   await expect(page).toHaveURL('/books/organizational-behavior/pages/preface')
 })
+
+test('signup', async ({ browserName, page }) => {
+
+  // GIVEN: Open Rex page
+  const BookPage = new ContentPage(page)
+  const path = '/books/introduction-intellectual-property/pages/1-5-what-the-u-s-patent-system-wrought'
+  await BookPage.open(path)
+
+  await rexUserSignup(page)
+  await expect(page).toHaveURL('/books/introduction-intellectual-property/pages/1-5-what-the-u-s-patent-system-wrought')
+  sleep(10000)
+
+  await BookPage.selecttext()
+
+  const notecard = page.locator("form[data-highlight-card='true']")
+  const blue =  notecard.locator("input[name='blue']")
+  await notecard.click()
+  // await blue.click()
+  await blue.setChecked(true)
```

Comment:
@gregfitch I am trying to highlight some text. Firefox works pretty fine but chrome and safari are having trouble finding colors in the highlighter. Can you please suggest what can be done